reference/OpenAPI.json

```diff
@@ -20686,9 +20878,13 @@
         },
         "description": "Request body schema for **POST** `/redemptions/{redemptionId}/rollback`."
```

Comment:
To chyba tez jest dla `redemptions` i `redemption`

Reply:
nie